pyproject.toml

```diff
@@ -29,7 +29,7 @@ exclude = '''
 github_url = "https://github.com/Amsterdam/schema-tools"
 
 [tool.tbump.version]
-current = "0.21.3"
+current = "0.21.4"
```

Comment:
tbump zou ik niet in jouw branch doen. Liever pas bij het uitbrengen van een nieuwe release (na review en merge).
in DSO-API kan je met `pip install -e ../schema-tools` ook testen of alles werkt.

Reply:
Version bump met `tbump` doe ik altijd na de review, maar vóór de _rebase and merge_ naar `master`